app/jobs/publish_feed_job.rb

```diff
@@ -0,0 +1,61 @@
+require 'builder'
+
+class PublishFeedJob < ApplicationJob
+
+  queue_as :feeder_default
+
+  include PodcastsHelper
+
+  attr_accessor :podcast, :episodes, :rss
+
+  def perform(podcast)
+    setup_data(podcast)
+    @rss = generate_rss_xml
+    save_podcast_file(@rss)
+  end
+
+  def setup_data(podcast)
+    @podcast = podcast
+    @episodes = @podcast.feed_episodes
+  end
+
+  def generate_rss_xml
```

Comment:
Ah, okay, so you're manually generating the xml in the worker, rather than calling out to the web to render the view.

Reply:
you got it.